doc/nvim-tree-lua.txt

```diff
@@ -390,6 +389,12 @@ Subsequent calls to setup will replace the previous configuration.
       notify = {
         threshold = vim.log.levels.INFO,
       },
+      ui = {
+        confirm = {
+          remove = true,
+          trash = true,
+        },
+      },
```

Comment:
Next time please remember to update help as per [CONTRIBUTING](https://github.com/nvim-tree/nvim-tree.lua/blob/master/CONTRIBUTING.md)

Added doc for this section and renamed to match the action.